src/app/app.component.ts

```diff
@@ -1,10 +1,29 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { BehaviorSubject, forkJoin, Observable, Subject, take, tap } from 'rxjs';
+import { AccountService } from './services/account.service';
+import { User } from './Users';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.less']
 })
-export class AppComponent {
-  title = 'Ascend';
+export class AppComponent implements OnInit {
+   currentUser$!:BehaviorSubject<User | string>; 
+   localStorageUser$!: BehaviorSubject<any>;
```

Comment:
Remove the use of `any` in your types. It eliminates the value of TypeScript because no types or property names can be checked. There is always a type you can provide, even if it is multiple types.

Reply:
I talked later about how this should be an Observable as well. We don't want to be working with Subjects in the app component. If you need access to the user it should come from an Observable in a service.

Reply:
This Subject has been made into an Observable.